static/scss/partials/main.scss

```diff
@@ -1087,10 +1087,6 @@ firefox-private-relay-addon[data-addon-installed="false"] + firefox-private-rela
         justify-content: space-between;
     }
 
-    .fx-private-relay-logotype {
```

Comment:
I'm not sure why this was set here in addition to in `header.scss`, but it seemed redundant, and removing it didn't change anything as far as I could see.